src/include/ipc/topology.h

```diff
@@ -47,6 +47,7 @@ enum sof_comp_type {
 	SOF_COMP_ASRC,		/**< Asynchronous sample rate converter */
 	SOF_COMP_DCBLOCK,
 	SOF_COMP_SMART_AMP,		/**< smart amplifier component */
+	SOF_COMP_CROSSOVER,
```

Comment:
I think this might be defunct now that UUID has landed

Reply:
Aren't types still used for matching component driver as fallback according to the UUID commit https://github.com/thesofproject/sof/pull/2920/commits/11a4a22042c6d83fa76e3e346da6129f622798fd ? Do we really want to skip it for crossover?

Reply:
Good point, this is generic, lets leave it

Reply:
@johnylin76 you are right for ABI < 3.17, but this will land for ABI >= 3.17 so it no longer needs this.